src/modules/stake-dot/components/Balance/Balance.tsx

```diff
@@ -0,0 +1,68 @@
+import { Paper, Typography, Button, Box, Tooltip } from '@material-ui/core';
+import React from 'react';
+import BigNumber from 'bignumber.js';
+
+import { t } from '../../../../common/utils/intl';
+import { ReactComponent as DOTIcon } from '../../assets/DOT.svg';
+
+import { useBalanceStyles } from './BalanceStyles';
+import { Body2 } from '../../../../UiKit/Typography';
+
+interface IBalanceProps {
+  amount: BigNumber;
+  isConnected?: boolean;
```

Comment:
```suggestion
  isConnected: boolean;
```
it doesn’t need to be optional when there is default value

Reply:
no, If I will not pass `isConnected` it will be an error from tslint. Default value set when the prop isConnected is undefined, so ii should point, that isConnected can be passed as undefined